code/src/components/Button.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react'
+
+export const Button = (props) => {
+  return (
+    <div className='submit-button-container'>
+      <button onClick={props.onClick} className='submit-button' type='submit' form=''>{props.title}</button>
+    </div>
```

Comment:
Instead of using {props.title} we should only write {title} and do:
`export const Button = ({title}) => {`

or like this:

`export const Button = props => {`
and:
`const {title} = props`